openbb_platform/core/openbb_core/provider/standard_models/balance_sheet.py

```diff
@@ -16,10 +16,6 @@ class BalanceSheetQueryParams(QueryParams):
     """Balance Sheet Query."""
 
     symbol: str = Field(description=QUERY_DESCRIPTIONS.get("symbol", ""))
-    period: str = Field(
-        default="annual",
-        description=QUERY_DESCRIPTIONS.get("period", ""),
-    )
```

Comment:
why are we removing `period` from this model?

Reply:
Because this is part of the issue, docstrings and type hints do not generate correctly. Long-standing issue being resolved.